addons.tf

```diff
@@ -202,3 +202,28 @@ module "cluster_secret_store" {
     module.addons
   ]
 }
+
+################################################################################
+# Reloader
+module "reloader" {
+  source = "./modules/addon"
+  create = var.create_addons && var.enable_reloader
+
+  chart         = "reloader"
+  chart_version = "1.0.121"
+  repository    = "https://stakater.github.io/stakater-charts"
+  description   = "Reloader"
+  namespace     = local.monitoring_namespace
+
+  create_namespace = true
+
+  # https://github.com/stakater/Reloader/blob/master/deployments/kubernetes/chart/reloader/values.yaml
+  values = [
+    file("${path.module}/files/helm/reloader/common.yaml")
```

Comment:
I dont think you need any change anything so this can be removed.

Reply:
Okay